README.adoc

```diff
@@ -683,8 +684,7 @@ mutations for Cloud Spanner. The dialect will only generate DML statements.
 
 === Locking
 
-Cloud Spanner does not support explicit locking clauses like `SELECT ... FOR UPDATE`. Setting an explicit lock level in
-Hibernate is therefore not supported. The following is for example not supported:
+Cloud Spanner supports exclusive row locks using https://cloud.google.com/spanner/docs/use-select-for-update[`SELECT ... FOR UPDATE`]
```

Comment:
```suggestion
Cloud Spanner supports explicit row locks using https://cloud.google.com/spanner/docs/use-select-for-update[`SELECT ... FOR UPDATE`]
```

These locks are not 'exclusive', as they do allow reads to continue to happen on these rows.